<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatNaviClass</class>
 <widget class="QMainWindow" name="MatNaviClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>854</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MatNavi</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/MatNavi/hustmap.bmp);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1700</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu_Q">
    <property name="title">
     <string>查询(Q)</string>
    </property>
    <addaction name="action_minChange"/>
    <addaction name="action_searchLoc"/>
   </widget>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_Q"/>
  </widget>
  <action name="action_minChange">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>最少转乘</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_searchLoc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>搜索地点</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_open">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>读入地图</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_exit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MatNavi.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_minChange</sender>
   <signal>triggered(bool)</signal>
   <receiver>MatNaviClass</receiver>
   <slot>minChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>843</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered(bool)</signal>
   <receiver>MatNaviClass</receiver>
   <slot>closewnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>843</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_open</sender>
   <signal>triggered(bool)</signal>
   <receiver>MatNaviClass</receiver>
   <slot>loadmap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>843</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_searchLoc</sender>
   <signal>triggered(bool)</signal>
   <receiver>MatNaviClass</receiver>
   <slot>searchLoc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>minChange()</slot>
  <slot>loadmap()</slot>
  <slot>closewnd()</slot>
  <slot>searchLoc()</slot>
 </slots>
</ui>
